client_consumer/pulp/client/consumer/config.py

```diff
@@ -1,7 +1,8 @@
 import os
 import socket
 
-from pulp.common.config import Config, REQUIRED, ANY, NUMBER, BOOL, OPTIONAL
+from pulp.common.config import Config, REQUIRED, ANY, NUMBER, BOOL, OPTIONAL, get_from_default_paths
```

Comment:
The jenkies tests are failing because of this import, I think it's the last one left to remove.